src/map/pc.c

```diff
@@ -1461,6 +1462,11 @@ int pc_reg_received(struct map_session_data *sd)
 		party->member_joined(sd);
 	if (sd->status.guild_id)
 		guild->member_joined(sd);
+	if (sd->status.clan_id)
+		clan->member_online(sd, true);
+
+	//Auth is fully okay, update last_login
+	sd->status.last_login = time(NULL);
```

Comment:
is this related to the clan system? If not please remove it and add it in another PR with relevant reasoning (since it is confusing).

Reply:
Yes it is, for removing inactive members.